README.md

```diff
@@ -1,4 +1,4 @@
-# MLonCode research playground [![PyPI](https://img.shields.io/pypi/v/sourced-ml.svg)](https://pypi.python.org/pypi/sourced-ml) [![Build Status](https://travis-ci.org/src-d/ml.svg)](https://travis-ci.org/src-d/ml) [![Docker Build Status](https://img.shields.io/docker/build/srcd/ml.svg)](https://hub.docker.com/r/srcd/ml) [![codecov](https://codecov.io/github/src-d/ml/coverage.svg)](https://codecov.io/gh/src-d/ml)
+# MLonCode research playground [![PyPI](https://img.shields.io/pypi/v/sourced-ml.svg)](https://pypi.python.org/pypi/sourced-ml) [![Build Status](https://travis-ci.org/src-d/ml.svg)](https://travis-ci.org/src-d/ml) [![Docker Build Status](https://img.shields.io/docker/build/srcd/ml.svg)](https://hub.docker.com/r/srcd/ml) [![codecov](https://codecov.io/github/src-d/ml/coverage.svg)](https://codecov.io/gh/src-d/ml) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```

Comment:
This readme is not about ml-core. For now, I suggest to create a new tiny README.md and make a proper one later. Let's keep Contribution, License, Glossary, Docker image. Algorithms have wrong links; Installation is different, so let's remove for now. 

I also suggest using `MLonCode Core Library` as a title and add a small relevant description. For example: 

```
Library for machine learning on the source code. Provides commonly used algorithms and tools to process the code-related data such as Babelfish's UASTs, plain code text, etc.
```
Feel free to change it as you like, it is an example.